src/component/panels/RangesPanel/TableColumns/ActionsColumn.jsx

```diff
@@ -61,34 +80,33 @@ const ActionsColumn = ({ rowData, onHoverSignal, onUnlink, rowSpanTags }) => {
 
   const saveEditRangeHandler = useCallback(
     (editedRange) => {
+      const _range = lodash.cloneDeep(editedRange);
       // for now: clear all assignments for this range because signals or levels to store might have changed
-      onUnlink();
-      // if all signals were deleted then insert a default signal with "m" as multiplicity
-      if (editedRange.signal.length === 0) {
-        addDefaultSignal(editedRange);
-        alert.info(
-          `There must be at least one signal within a range. Default signal with "m" was therefore added!`,
-        );
-      }
+      unlinkInAssignmentData(_range);
+      unlink(_range);
+
+      delete _range.tableMetaInfo;
+
       dispatch({
         type: CHANGE_RANGE_DATA,
-        data: editedRange,
+        data: _range,
       });
     },
-    [alert, dispatch, onUnlink],
+    [dispatch, unlinkInAssignmentData],
   );
 
   const closeEditRangeHandler = useCallback(() => {
+    dispatch({ type: RESET_SELECTED_TOOL });
     modal.close();
-  }, [modal]);
+  }, [dispatch, modal]);
 
   const openEditRangeHandler = useCallback(() => {
-    dispatch({ type: RESET_SELECTED_TOOL });
+    dispatch({ type: SET_SELECTED_TOOL, selectedTool: 'editRange' });
     modal.show(
       <EditRangeModal
-        onClose={closeEditRangeHandler}
-        onSave={saveEditRangeHandler}
-        onZoom={zoomRangeHandler}
+        onCloseHandler={closeEditRangeHandler}
+        onSaveHandler={saveEditRangeHandler}
+        onZoomHandler={zoomRangeHandler}
         range={rowData}
```

Comment:
you have to change the event name to be like as the same as standards and not use word handler as part of the event name you can you use it in event handler name

Reply:
Just as a remark: I can not name it as the usual "onClose" because then the ModalProvider's "closeHandler" passed as "onClose" to the modal itself method is called instead